build.ps1

```diff
@@ -24,6 +24,8 @@ try {
 
     Write-Host "Building project..."
     gulp build $buildArgs
+
+    nuget pack -OutputDirectory ..\dist

```

Comment:
if nuget.exe is not in Path, this won't work. Please add error handling or update build instruction so that users update Path environment variable to include the nuget.exe path.

Reply:
The output directory is misleading users because it uses relative path. Please use the full path to avoid any confusion.

Reply:
I think "nuget pack" should be called only when the --wac is set because the nuget package is only used by WAC.

Reply:
added more input parameters and checking logic that should resolve these issues (next commit)